argo-cd/Exposed_UI/README.md

```diff
@@ -0,0 +1,23 @@
+# setup anonymous admin access for all versions 
+
+```bash
+kubectl create namespace argocd
+kubectl apply -n argocd -f https://raw.githubusercontent.com/argoproj/argo-cd/stable/manifests/install.yaml
+kubectl edit -n argocd cm argocd-cm -o yaml
+```
+
+append the following:
+```yaml
+data:
+  users.anonymous.enabled: "true"
+```
+
+`kubectl edit -n argocd cm argocd-rbac-cm -o yaml`
+
+append the following:
+```yaml
+data:
+  policy.default: role:admin
+```
```

Comment:
Please consider making a simple shell build script for ease of use. It should be enough to replace the two `kubectl edit` interactive commands with their automated counterparts:

- `kubectl patch -n argocd cm argocd-cm -p '{"data":{"users.anonymous.enabled":"true"}}'`
- `kubectl patch -n argocd cm argocd-rbac-cm -p '{"data":{"policy.default":"role:admin"}}'`